dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
I guess that new method is part of a environment vendor API rather than a client API but we haven't separated them.